src/python/pants/core/goals/test.py

```diff
@@ -140,6 +140,9 @@ def message(self) -> str:
             output = f"{output.rstrip()}\n\n"
         return f"{message}{output}"
 
+    def metadata(self) -> Optional[Dict[str, Any]]:
+        return {"address": self.address }
```

Comment:
Black won't like the extra space.

```suggestion
        return {"address": self.address}
```

Reply:
Should this be self.address.spec so that it's a string? That's a much nicer representation.